pages/5g-media-streaming/tutorials/metrics-reporting.md

```diff
@@ -139,7 +139,26 @@ variable with the value from the JSON file:
 
 In Postman navigate to `Collections` and select `5G-MAG M1`. Navigate to `Metrics-Reporting` and
 select `Create Metrics Reporting Configuration`. Then adjust the attributes in the payload section based on the desired
-configuration. Click on `Send` once the configuration is set:
+configuration. An example configuration looks the following:
+
+````json
+{
+   "scheme": "",
```

Comment:
I don't think an empty _scheme_ property is valid here.

Either the property should be populated with a valid URI or else it should be omitted altogether.